training/README.md

```diff
@@ -88,6 +88,38 @@ print("Pred text:", pred_text)
 print("Environment set up successful?", generated_ids.shape[-1] == 20)
 ```
 
+## 0. Recommendations and guidelines 
```

Comment:
Should we put this section at the end of the README? i.e. before https://github.com/huggingface/distil-whisper/tree/main/training#overview-of-training-methods

If we do this, we can assume the user has read the entire README once end-to-end, and so the section content will make more sense

Reply:
Yes, I agree, this way mentions to pseudo labelling etc will be clearer!